api/StyleSheetList.json

```diff
@@ -124,10 +124,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "10"
```

Comment:
This is a `length` property and I don't trust the results without testing an instance.

Reply:
Ack, didn't mean to create this PR...